cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -157,6 +157,10 @@ Resources:
                 Value: !Ref 'FilesystemTagValue'
               - Key: aws-parallelcluster-node-type
                 Value: Compute
+    {%- for tag in tags %}
```

Comment:
While addressing this shall we also duplicate the Tags array and assign it to a ResourceType of type volume so that hopefully we also get EBS volumes tagged correctly?

Reply:
Can you include tags in the input provided in test_hit_substack_rendering (`cloudformation/tests/test_cfn_template_rendering.py`)?

Reply:
I believe both of these have been addressed.